README.md

```diff
@@ -5,86 +5,115 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/canonical/microceph/microceph)](https://goreportcard.com/report/github.com/canonical/microceph/microceph)
 [![Documentation Status](https://readthedocs.com/projects/canonical-microceph/badge/?version=latest)](https://canonical-microceph.readthedocs-hosted.com/en/latest/?badge=latest)
 
-<p align="center">
-<a href="https://snapcraft.io/microceph">MicroCeph</a> is snap-deployed Ceph with built-in clustering.
-</p>
-
 [![Get it from the Snap Store][snap-button]][snap-microceph]
 
-## Table of contents
 
-* [💡 Philosophy](#-philosophy)
-* [🎯 Features](#-features)
-* [⚡️ Quickstart](#%EF%B8%8Fquickstart)
-* [📖 Documentation](#-documentation)
-* [💫 Project & community](#-project--community)
-* [📰 License](#-license)
+MicroCeph is an opinionated orchestration tool for Ceph clusters at all scales.
+It reduces the complexity of deploying and managing clusters by simplifying various operations like, service placement,
+disk administration and remote replication via RESTful APIs and CLI commands.
+
+Available as a snap, MicroCeph is the easiest tool for administrators, developers, and hobbyists to manage clusters.
```

Comment:
`manage clusters` -> `manage ceph clusters`

Reply:
Nit: `manage Ceph clusters`

Reply:
@skoech I think this suggestion hasn't made it into the updates yet -- it's not a blocker for me just want to ensure it's intentional

Reply:
Done. I had added "Ceph" to the previous line (...managing clusters...).